src/App.js

```diff
@@ -1,16 +1,66 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import ColorChoice from './components/ColorChoice';
 
 const App = () => {
+  const [messages, setMessages] = React.useState(chatMessages);
+
+  const likeMessage = (id) => {
+    setMessages(
+      messages.map((message) => (message.id === id ? {
+        ...message,
+        liked: !message.liked
+      } : message))
+    );
+  };
```

Comment:
This is an optimal approach to calculating the number of likes, great job 😁

This is a great place to calculate & display the total likes